Parsers/ASimFileEvent/Parsers/vimFileEventSentinelOne.yaml

```diff
@@ -0,0 +1,109 @@
+ParserName: vimFileEventSentinlOne
+EquivalentBuiltInParser: _Im_FileEvent_SentinelOne
+ParserQuery: |
+  let ASIM_GetWindowsFilenamePart = (path: string) { tostring(split(path, @'\')[-1]) };
+  let ASIM_GetLinuxFilenamePart = (path: string) { tostring(split(path, @'/')[-1]) };
+  let EventTypeLookup = datatable (alertInfo_eventType_s: string, EventType: string)
+      [
+      "FILECREATION", "FileCreated",
+      "FILEMODIFICATION", "FileModified",
+      "FILEDELETION", "FileDeleted",
+      "FILERENAME", "FileRenamed"
+  ];
+  let parser = () {
+      let allFileData = SentinelOne_CL
+          | where event_name_s == "Alerts."
+              and alertInfo_eventType_s in ('FILECREATION', 'FILEMODIFICATION', 'FILEDELETION', 'FILERENAME');
+      let windowsFileData = allFileData
+          | where agentDetectionInfo_osFamily_s == "windows"
+          | extend
+              TargetFilePathType = "Windows Local",
+              TargetFileName = ASIM_GetWindowsFilenamePart(targetProcessInfo_tgtFilePath_s),
+              SrcFileName = ASIM_GetWindowsFilenamePart(targetProcessInfo_tgtFileOldPath_s);
+      let otherFileData = allFileData
+          | where agentDetectionInfo_osFamily_s != "windows"
+          | extend
+              TargetFilePathType = "Unix",
+              TargetFileName = ASIM_GetLinuxFilenamePart(targetProcessInfo_tgtFilePath_s),
+              SrcFileName = ASIM_GetLinuxFilenamePart(targetProcessInfo_tgtFileOldPath_s);
+      union windowsFileData, otherFileData
+      | lookup EventTypeLookup on alertInfo_eventType_s
+      | extend
+          EventSeverity = iff(ruleInfo_severity_s == "Critical", "High", ruleInfo_severity_s),
+          EventVendor = "SentinelOne",
+          EventProduct = "SentinelOne",
+          EventResult = "Success",
+          EventSchema = "FileEvent",
+          EventSchemaVersion = "0.2.1",
+          ActorUsername = coalesce(sourceProcessInfo_user_s, "N/A")
+      | project-rename
+          EventStartTime = sourceProcessInfo_pidStarttime_t,
+          EventOriginalSeverity = ruleInfo_severity_s,
+          EventUid = _ResourceId,
+          ActingProcessCommandLine = sourceProcessInfo_commandline_s,
+          ActingProcessGuid = sourceProcessInfo_uniqueId_g,
+          ActingProcessId = sourceProcessInfo_pid_s,
+          ActingProcessName = sourceProcessInfo_name_s,
+          DvcId = agentDetectionInfo_uuid_g,
+          DvcOs = agentDetectionInfo_osName_s,
+          DvcOsVersion = agentDetectionInfo_osRevision_s,
+          EventOriginalType = alertInfo_eventType_s,
+          EventOriginalUid = alertInfo_dvEventId_s,
+          RuleName = ruleInfo_name_s,
+          TargetFileCreationTime = targetProcessInfo_tgtFileCreatedAt_t,
+          SrcFilePath = targetProcessInfo_tgtFileOldPath_s,
+          TargetFilePath = targetProcessInfo_tgtFilePath_s,
```

Comment:
can you validate SentinelOne filepath is matching with normalization recommendations?
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-file-event#path-structure

Reply:
Done. Validated.